src/pkcs15init/profile.c

```diff
@@ -1985,24 +1985,40 @@ static struct block	root_ops = {
    "root",		process_block,	NULL,		root_blocks
 };
 
+static void
+get_inner_word(char *str, char word[64]) {
+	char *inner = NULL;
+	size_t len = 0;
+
+	inner = str;
+	while (isalnum((unsigned char)*inner) || *inner == '-' || *inner == '_') {
+		inner++;
+		len++;
+	}
+	len = len >= 64 ? 63 : len;
```

Comment:
using `sizeof(word)` instead of magic number would it make a bit more readable.

Reply:
This would return only size of `char *`, but will use macro instead, thanks.